yt_dlp/extractor/iprima.py

```diff
@@ -131,6 +131,7 @@ def _real_extract(self, url):
         video_id = self._search_regex((
             r'productId\s*=\s*([\'"])(?P<id>p\d+)\1',
             r'pproduct_id\s*=\s*([\'"])(?P<id>p\d+)\1',
+            r'let videos\s*=\s*([\'"])(?P<id>p\d+)\1',
```

Comment:
```suggestion
            r'let\s+videos\s*=\s*([\'"])(?P<id>p\d+)\1',
```

Reply:
In 2be0239ee18c9b3414e3d232035338bc5a8f3b69, the a testcase for zoom.iprima.cz added.